Solutions/MicrosoftDefenderForEndpoint/Playbooks/Restrict-MDEUrl/readme.md

```diff
@@ -18,6 +18,12 @@ After deployment, you can run this playbook manually on an alert or attach it to
 
```

Comment:
1. Correct the spelling 'rune'.
2. In readme file Image links are broken and entity trigger screenshot is missing.